servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpQuery.java

```diff
@@ -115,7 +115,7 @@ public HttpQuery add(final String key, final String... values) {
     }
 
     public HttpQuery set(final String key, final String value) {
```

Comment:
Do we need the `@Nullable` annotation on `value`? Samve with `add` above.

Reply:
yes good call, I realized this too after I pushed the PR. done now.